src/routes/resources/joining/+page.svelte

```diff
@@ -12,31 +13,44 @@
 		resources to opportunities for growth. Your participation not only enriches your experience but
 		also helps our chapter give resources and expeience that you can use. We are diligently working
 		towards inproving all student's education and experience, and your involvement is instrumental
-		in this endeavor.
-	</p>
-	<p>
-		If you are interested in joining ACM you can simply attend one our meetings or talk to anyone
-		that you know is associated with ACM about joining. If our meeting times don't work with you,
-		you can still participate in ACM with our subcommittees.
-	</p>
-	<p>
-		<b>Membership dues are $20 for one school year.</b> They're are not required but it does help out
-		the club to afford T-shirts and equipment.
-	</p>
-	<p>
-		Please make a myACM account, it's free and it'll continue supporting our chapter. <b>
-			Use this link to sign up:</b
-		>
-		<a
-			href="https://accounts.acm.org/"
-			class="text-blue-900 font-bold underline hover:text-blue-700"
-			>myACM
-		</a>
-	</p>
-	
-	<p>
-		Once you have successfully made an account, please forward your ACM ID number to either the
-		President or Secretary. Your contribution is pivotal in helping us achieve our membership goals
-		and further our impact as a chapter.
+		in this endeavor. <span>Membership dues are $20 for one school year.</span> They are not required
+		but it does help the club afford T-shirts and equipment.
 	</p>
+
+	<div class="py-4 grid gap-x-4 gap-y-20 md:grid-cols-2 sm:gap-y-16">
+		<div>
+			<p class="text-xl font-bold text-gray-900 sm:text-xl">Steps to join <span>WSC ACM</span>:</p>
+			<br>
+			<List class="list-decimal space-y-4">
+				<Li>
+					Create an account with <a href="https://accounts.acm.org/">myACM</a>: It’s free and gives
+					you access to <span>ACM</span> resources and opportunities.
+				</Li>
+				<Li>
+					Send your <span>ACM ID</span> to the <span>President</span> or
+					<span>Vice President</span>: This allows us to add you to the <span>WSC ACM</span> chapter,
+					to continue as a club on campus we need at least 20 registered members.
+				</Li>
+				<Li>
+					Attend our <span>ACM</span> meetings: Check the
+					<a href="{base}/meetingtimes">Meeting Times</a>
+					page for more information. If our meeting times don't work with you, you can still
+					participate in <span>ACM</span> with our subcommittees.
+				</Li>
+			</List>
+		</div>
+		<div class="flex items-center">
```

Comment:
put this more to the right so theres not that awkward space on the right of the gif. The distance from the left side of the text should be the same with the gif

Reply:
![Screenshot 2024-10-29 021551](https://github.com/user-attachments/assets/0478eaec-e768-44c9-83b7-106193460d4e)
Not sure of the space you're talking about, check this screenshot out

Reply:
I like that yes, but we have to think about all screen sizes

Reply:
![Screenshot 2024-10-29 172933](https://github.com/user-attachments/assets/df337615-ded4-417c-8217-f3d91453c7d7)
 also if you can for me, go to lib/components/ScreenNav.svelte and add in an extra period on line 21 inside the src quotes in front of the slash. The image in he top left isn't showing up
![image](https://github.com/user-attachments/assets/d6fb4df2-dfd2-4ab7-a4b3-4162dcd2fd7e)

Reply:
done 👍